src/main/java/lottery/domain/StaticPicker.java

```diff
@@ -0,0 +1,18 @@
+package lottery.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class StaticPicker implements Picker {
+    private final String[] numbers;
+
+    public StaticPicker(String input) {
+        numbers = input.split(" *, *");
+    }
+
+    @Override
+    public List<Integer> pick() {
+        return Arrays.stream(numbers).map(Integer::parseInt).collect(Collectors.toList());
+    }
```

Comment:
```suggestion
    public List<Integer> pick() {
        return Arrays.stream(numbers)
                .map(Integer::parseInt)
                .collect(Collectors.toList());
    }
```
스트림 메소드는 한 줄에 한 개만 사용해주세요!

Reply:
수정: 63f67e4